parts/kubernetesagentvars.t

```diff
@@ -1,7 +1,9 @@
     "{{.Name}}StorageAccountOffset": "[mul(variables('maxStorageAccountsPerAgent'),variables('{{.Name}}Index'))]",
     "{{.Name}}Count": "[parameters('{{.Name}}Count')]",
+    "{{.Name}}Offset": "[parameters('{{.Name}}Offset')]",
+    "{{.Name}}TotalCount": "[add(parameters('{{.Name}}Count'), parameters('{{.Name}}Offset'))]",
```

Comment:
agreed it is in my list of things todo to fix this PR up and get it ready for checkin